### ISSUE TITLE
(PRE) Reviewer 2, second report

### ISSUE THREAD

**Issue by labarba** [barbagroup/pygbe_lspr_paper#13]
Previous comments and responses:
https://github.com/barbagroup/pygbe_lspr_paper/issues/11

It is fair the authors thought that some of the references are not quite related to their current study. But it does not make sense to say that the quantum effect is beyond the scope of the research so they do not care about it. To be more specific the authors should justify why their full classical calculation at distance $d=0.5~$nm makes any sense --- namely why we should believe it is right. This is extremely important and relevant here: the authors should make it clear when their prediction is reliable since the whole research here is about an in-house numerical soft-ware/method. And I strongly doubt that at $d=0.5~$nm there will be (quantum) tunneling of electrons between the metallic particle and the molecules; if this is the case then the quantitative analysis at such a small distance does not make much sense. So there are two choices: 1) the authors could argue that the quantum effect is negligible so their result is correct; or 2) the authors delete the relevant result to make their conclusion scientifically sound.

As a result I would not recommend the acceptance of current manuscript for publication until the authors address the above comment properly.

**Comment by labarba** [comment]
**Comment**
The reviewer is concerned that the case with smallest distance (d=0.5 nm) could be incorrect, due to the fact that our model does not include quantum effects. The question of whether quantum effects, e.g., quantum tunneling of electrons between the nanoparticle and the protein, are applicable in this case is not straightforward to answer. We reviewed the literature more carefully to answer the reviewer's question, and found the following:

**Ideas**
- The paper of [Savage et al 2012](https://www.nature.com/articles/nature11653.pdf) study the quantum regime where tunnelling occurs when two metal nanostructures are placed few nanometers apart. In their paper the authors mention that for `d>~0.4 nm`, plasmons interactions are consistent with the classical approach. They state that the crossover where the quantum and classical predictions diverge occurs at `d=0.31nm`. For `d<0.31nm` the quantum tunelling increases exponentially and quickly dominates, while  For `d>0.31` the mention that "the spectra are dominated by the near-field interaction of the cavity-localized surface charges and plasmons couple according to classical models".  
In their article they also mention that fabrications that have been able to achieve gaps as small as `0.5 nm` between the nanoparticles, fail to reach the quantum tunnelling regime.  

- The paper of [Esteban et al 2012](https://www.nature.com/articles/ncomms1806.pdf) presents a "quantum corrected model", that incorporates quantum- mechanical effects in the classical electrodynamics approach. They show tunnelling transmission probability  as a function of separation, they state that the tunnelling regime happens between `0.1 nm < d <~0.5 nm`. However,by looking at their presented results the probability of transmission at 0.5 nm is almost zero.  

We understand that depending on the literature, a distance of `d=0.5 nm` might be on the limit where quantum effects start to be present [Garcia de Abajo 2008](https://pubs.acs.org/doi/pdf/10.1021/jp807345h?rand=q4g7gkca), [Ciraci et al 2012](https://science.sciencemag.org/content/sci/337/6098/1072.full.pdf). But other papers in the field, have perform similar studies to ours (having small separations between the surfaces) where the problems have been modeled by using only classical approaches. For example, [Grillet et al 2011](https://pubs.acs.org/doi/pdf/10.1021/nn2041329?rand=cdjw4js5) study the coupling in silver nanocubes at distances as small as 0 nm using Discrete Dipole approximation and Mie theory. The work of [Mcmahon et al 2009](https://link.springer.com/content/pdf/10.1007%2Fs00216-009-2738-4.pdf) studies electromagnetic enhancement on gold nanoparticle dimers using finite element calculations that do not include quantum effects, for distances between the dimmers as small as 0.25 nm. 

After reviewing the literature we concluded that it is not clear if the distance `d=0.5 nm` is or not into a quantum regime, since it is not clear how the results on the papers cited extrapolate to our particular case. However, we understand the concerns of the reviewer and we decided to address this on the manuscript by explaining that we are aware of the possibilities of quantum effects but that there is not enough evidence on the literature to claim if they are present or not in our  particular case, therefore we leave the interpretation at the discretion of the reader. 

**Reply**
The reviewer raises an important point in this comment. It is true that our model cannot handle quantum effects, such as tunneling, which may be present at small distances between the analyte and the nanoparticle. In particular, this may be a problem for our last test case, when that distance is 0.5nm.

The literature gives some evidence that 0.5nm is within the validity domain of a classical approach:
-  [Savage et al 2012](https://www.nature.com/articles/nature11653.pdf) studied tunneling effects in plasmonic systems. In particular, they consider two nanostructures, and look at tunneling as a function of the inter particle distance. In this paper, the authors claim that plasmon interactions are consistent with the classical approach when `d>~0.4nm`. Moreover, their results show that the quantum and classical predictions start diverging at `d=0.31nm`. 
- [Esteban et al 2012](https://www.nature.com/articles/ncomms1806.pdf) presents a "quantum corrected model", that incorporates quantum-mechanical effects in the classical electrodynamics approach. Their results show the tunnelling transmission probability  as a function of separation between two metallic nanospheres, and state that the tunnelling regime happens between `0.1 nm < d <~0.5 nm`. However, the probability of transmission at 0.5 nm is almost zero.  

These two articles support that, in systems that are similar to ours, quantum effects can be ignored for the one case we include with `d=0.5nm`. However, it is close to the limit. We are aware of articles that consider such distances inside the quantum regime, for example, [Garcia de Abajo 2008](https://pubs.acs.org/doi/pdf/10.1021/jp807345h?rand=q4g7gkca) and [Ciraci et al 2012](https://science.sciencemag.org/content/sci/337/6098/1072.full.pdf)

In conclusion, we understand the reviewer's concern, however, from the literature it is not clear that for the distance `d=0.5 nm` quantum effects are significant. In response to the reviewer's concern, we added a disclaimer stating that the model does not consider quantum effects, even though they might be present in our last test case. We mention it both in the caption of Fig 11 as well as in the Discussion section, with references.

**Modifications:**
- Add note in caption of Fig 11: commit number 13878d3 and 6965bbd 
- Add comment on discussion section: commit number 178e02f, 6998713 and 567a395
